README.md

```diff
@@ -771,6 +771,12 @@ based session store. Supports all backends supported by Fortune (MongoDB, Redis,
 
 [![★][restsession-image] restsession][restsession-url] Store sessions utilizing a RESTful API
 
+
+[prisma-session-store-url]: https://www.npmjs.com/package/@quixo3/prisma-session-store
+[prisma-session-store-image]: https://badgen.net/github/stars/kleydon/prisma-session-store?label=%E2%98%85
+
+[![★][prisma-session-store-image] prisma-session-store][prisma-session-store-url] Express session store for the [Prisma Framework](https://www.prisma.io).
```

Comment:
It does look like during the [amd based builds node 6 onwards](https://travis-ci.org/github/expressjs/session/jobs/700400146)  there is an issue with the links.

Reply:
Linting only runs on Node.js 6+; the changes simply fail the linting. Run `npm run lint` locally to lint README.

Reply:
Hi @dougwilson - thanks for considering my PR.

Not entirely sure what to do here. I tried npm run lint locally (in the package directory), and got a bunch of errors re: indentation for the file session/test/session.js, but no errors for the one file that I modified: README.md.

I _did_ find one spacing difference, which I've fixed in my latest commit, but other than that, the format for the links I've added seems to be exactly the same as the format for the surrounding links in the Readme. I've verified that the links I added aren't dead links.

Any additional ideas for how to approach any remaining resolution that needs to happen?

Reply:
You inserted your prisma session in the middle of the restsession section.

Reply:
>> I'm going to have a little look at prisma. Just a bit of due diligence, as I have not used this before.

It seems promising (if still rough around the edges) - particularly for folks who are comfortable with javascript / typescript, but less comfortable with databases.

Reply:
>>You inserted your prisma session in the middle of the restsession section.

Good point! I can see how that might be a problem.
Just committed a fix for this...